tutorials/3_instrument_resource_and_service/README.md

```diff
@@ -0,0 +1,128 @@
+[![Experimental Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Experimental.png)](https://opensource.newrelic.com/oss-category/#experimental)
+
+# Demo Catalog : Tutorial : Instrument a Resource and Service
+
+If you have not got the demo-deployer setup please follow our [getting started guide](/GETTING_STARTED.md) and get your AWS and New Relic credential setup before working through this tutorial.
+
+# Tutorial : How to build a demo: Instrumentation a resource and service
+
+In this third tutorial we will be creating a deploy configuration that will provision a single resource, install a service on that resource and setup New Relic instrumentation on both.  All instrumentation is defined in a block called "instrument" within a deployment configuration JSON file.  The demo-deployer will install and setup the New Relic instrumentation using the user configuration credentials you provided in the demo-deployer setup.
```

Comment:
`All instrumentation is defined in a block called "instrument" within a deployment configuration JSON file.` should be update to `All instrumentation is defined in a block call "instrumentation" within a deployment configuration JSON file.`